WikiClone/Article.html

```diff
@@ -6,65 +6,69 @@
 </head>
 <body>
 <header>
-	<ul class="header_navbar">
-		<li class="navbar_item"><a href="Main.html"><p class="title">WikiClone</p></a></li>
-		<li class="navbar_item" style="float:right"><a href="Register.html" class="header_link">Register</a></li>
-		<li class="navbar_item" style="float:right"><a href="Login.html" class="header_link">Login</a></li>
-	</ul>
+	<div class="header_navbar">
+		<a href="Main.html"><p class="title">WikiClone</p></a>
+		<div class="nav">
+			<div class="navbar_item"><a href="Login.html" class="header_link">Login</a></div>
+			<div class="navbar_item"><a href="Register.html" class="header_link">Register</a></div>
+		</div>
+	</div>
 </header>
 <main>
 	<div class="content">
 		<div class="container_name">
-			<div class="container">
+			<div class="title_navbar">
 				<div class="item_left">
 					<h1>
 						Legionary
 					</h1>
 				</div>
-				<div class="item_right">
-					<a class="button_site">Remove</a>
-				</div>
-				<div class="item_right">
-					<a href="EditArticle.html" class="button_site">Edit</a>
-				</div>
+				<nav>
+					<div class="item_right">
```

Comment:
And the layout is broken now 
![image](https://user-images.githubusercontent.com/138737/117337313-e46b3480-aea5-11eb-8357-97da54c2ba44.png)

Reply:
Fixed it everywhere